Doc/whatsnew/3.7.rst

```diff
@@ -2120,8 +2120,9 @@ Platform Support Removals
   of other LTS Linux releases (e.g. RHEL/CentOS 7.5, SLES 12-SP3), use OpenSSL
   1.0.2 or later, and remain supported in the default build configuration.
 
-  CPython's own :source:`CI configuration file <.travis.yml>` provides an
-  example of using the SSL
+  CPython's own `CI configuration file
+  <https://github.com/python/cpython/blob/5fe7c98a54d127759cfce323fab831008c945964/.travis.yml>`_
```

Comment:
I wonder if it makes more sense to use a tag instead of the hash of the original commit, e.g. https://github.com/python/cpython/blob/v3.7.10/.travis.yml

For instance, it looks like there have been several backports to the travis file, and this way the link makes it clearer what exactly you're looking at.

Reply:
I have made the requested changes; please review again.